src/views/admin/RolleCreationView.vue

```diff
@@ -84,7 +88,24 @@
       })
       .join(', ')
   })
-  onMounted(() => {
+
+  // TODO: As soon as we have a unique combination of attribute besides the UUID (org.id)
+  // it should be used as a title here for UI as a user friendly display
+  const organisationsForSelect: ComputedRef<
+    {
+      value: string
+      title: string
+    }[]
+  > = computed(() =>
+    organisationStore.AllOrganisations.map((org: Organisation) => ({
+      value: org.id,
+      title: org.id
```

Comment:
```suggestion
      title: org.name
```

Reply:
Hier bleiben wir bei org.id, da der Name nicht unique ist und wir den Fall haben, dass wir mehrmals den gleichen Namen als title haben in der UI.

Reply:
Wird nochmal besprochen mit Christian und Philipp nächste Woche aber das ist die einzige Lösung, die Sinn macht ohne Dienstellenummer (haben wir noch nicht).

Reply:
Ich habe es zum einen an die falsche Stelle kommentiert, der item-text im Template. muss der title sein. Aber: Das bringt nichts, wenn der title hier die id ist. Mit den IDs können die User nichts anfangen.

Reply:
Wenn es eh nur eine Übergangslösung ist, dann kann auch der Name rein. Und langfristig dann "Name (Dienststellennummer)"

Reply:
Richtig, das war genau mein Punkt an Christian heute morgen... die Alternativen sehen aber auch nicht gut aus... Der "name" ist nicht unique, der "Typ" ist nicht unique.. Die Kombination name + typ ist auch nicht unique. 

Wir kriegen in der Ergebnisseite sowieso nur die Id der Organisation... Also wenn wir in der Form ein "name" zeigen und in der Ergebnisseite dann die Id davon sieht das auch schlecht aus. Ich und Kristoff sehen wir das so: Solange wir kein Dienstellennummer vom Backend bekommen, zeigen wir die Id in der UI (Zumindest können wir damit die UI testen). 

Nächste Woche besprechen wir das sowieso.

Reply:
Ergänzung: Nach heutigem Gespräch mit PO und Architekten haben wir uns entschieden den Dropdown so einzustellen:
Dienstellennummer (Name). In der Ergebnisseite muss noch Kristoff gucken, wie er uns die Daten liefert, da wir jetzt nur die Id in der Response bekommen.